Development environment setup.md

```diff
@@ -0,0 +1,34 @@
+## Install basic tools
+
+- [Latest Oracle JDK](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Android Studio](https://developer.android.com/studio/index.html)
+- Git—`brew install git` ([brew docs](http://brew.sh/))
+- Zeplin—`brew cask install zeplin` ([brew cask docs](https://caskroom.github.io/))
+
+## Import code style
+
+To make working with the rest of the team easier, we all use the same code style settings.
+This ensures there are no unnecessary changes caused by code style differences and also makes it easier to conform to the established ckeckstyle rules. The code style includes both Java and XML style.
```

Comment:
We need to also add Kotlin here since we have some changes in style for the new language.